colin-api/src/colin_api/resources/business.py

```diff
@@ -170,3 +170,31 @@ def get(info_type, legal_type=None, identifier=None):  # pylint: disable = too-m
         except Exception as err:  # pylint: disable=broad-except; want to catch all errors
             current_app.logger.error(err.with_traceback(None))
             return jsonify({'message': 'Something went wrong.'}), HTTPStatus.INTERNAL_SERVER_ERROR
+
+
+@cors_preflight('GET')
+@API.route('<string:corp_num>', methods=['GET'])
```

Comment:
This path doesn't work.  

Update to `@API.route('/<string:corp_num>', methods=['GET'])`.

Was missing the "/" at the front of the route.

Reply:
I updated